lib/index.ts

```diff
@@ -155,8 +166,16 @@ const nodeSdk = ({ managementKey, publicKey, ...config }: NodeSdkArgs) => {
         await sdk.validateJwt(refreshToken);
         const jwtResp = await sdk.refresh(refreshToken);
         if (jwtResp.ok) {
-          const token = await sdk.validateJwt(jwtResp.data?.sessionJwt);
-          if (jwtResp.data.refreshJwt) {
+          // if refresh was successful, validate the new session JWT
+          const seesionJwt =
+            getCookieValue(
+              (jwtResp.data as JWTResponseWithCookies)?.cookies?.join(';'),
```

Comment:
Just a personal opinion, joining cookies seems to me a bit error prone , I would create a func to search for each cookie instead join, but this way is ok as well